test/config/examples/config_sharedstorage_trial.py

```diff
@@ -0,0 +1,24 @@
+"""
+A deep MNIST classifier using convolutional layers.
+
```

Comment:
If we don't use the nni example, I think create a new folder to handle trial code and config parallel with `test/config/examples` is better.

Reply:
Fixed